.github/workflows/app.yaml

```diff
@@ -0,0 +1,23 @@
+name: "All the sdk specific jobs"
+
+on:
+  workflow_call:
+
+jobs:
+  pub-dev-dry-run:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: subosito/flutter-action@48cafc24713cca54bbe03cdc3a423187d413aafa
+        with:
+          channel: stable
+          cache: true
+      - name: Prepare publish
+        run: ./package_flutter.sh
+      - name: Check podfile
+        run: cat publishing/flutter/ios/flutter_olm.podspec
```

Comment:
unrelated?

Reply:
I have copied it from other repo. Removed